database/db.js

```diff
@@ -12,21 +12,54 @@ db.once('open', function() {
   console.log("connected to the DB: ")
   });
 
-
-
-  
-
-
-const imagemodal = new Schema({
+const imageModal = new Schema({
   restaurauntTitle: String,
   helpfull: Number,
   notHelpfull: Number,
   images: Array
-
 });
 
-//use for reference in seeding script
+const getImages = () => {
+  imageModal.find()
+    .then(console.log('found images'))
+    .catch(console.log('cant find images'))
+}
+
+const getImageById = (id) => {
+  imageModal.findById(id)
+    .then(console.log(`found image by ${id}`))
```

Comment:
You are going to want to send the image data back to the module that made the request.